test/oauth-server/service/service.go

```diff
@@ -19,27 +20,39 @@ type Service struct {
 
 // New parses configuration and creates new Server with provided store and bus
 func New(ctx context.Context, config Config, logger log.Logger) (*Service, error) {
+	otelClient, err := otelClient.New(ctx, config.Clients.OpenTelemetryCollector, "oauth-server", logger)
+	if err != nil {
+		return nil, fmt.Errorf("cannot create open telemetry collector client: %w", err)
+	}
+	//tracerProvider := otelClient.GetTracerProvider()
```

Comment:
Remove? Is the otelClient active without this? In other services the tracerProvider is provided to the `New` function.

Reply:
I just leave it as for the future when I integrate otel to HTTP servers.